spec/unit/jobs/services/service_instance_state_fetch_spec.rb

```diff
@@ -387,6 +387,66 @@ def run_job(job)
                 end
               end
             end
+            context 'when exponential backoff is not set to default' do
+              before do
+                TestConfig.config[:broker_client_async_poll_exponential_backoff_rate] = 2.0
+              end
+
+              describe 'it updates the polling interval' do
```

Comment:
`context` (and `describe`) should be used to indicate a test scenario (e.g. `context 'when some value is configured'`), `it` describes the expectation (e.g. `it 'returns another value'`). I know that there are files where this has not been done consistently, but this spec adheres to this convention. Thus my proposal is to structure the new tests as follows:
```
context 'when exponential backoff is not set to default' do
  it 'calculates the polling intervals based on the default interval and the exponential backoff rate' do
    # set broker_client_async_poll_exponential_backoff_rate
    # ...
  end
 it 'calculates the polling intervals based on the configured interval and the exponential backoff rate' do
    # set broker_client_async_poll_exponential_backoff_rate + broker_client_default_async_poll_interval_seconds
    # ...
  end
 it 'takes the exponential backoff into account when checking whether the next run would exceed the maximum duration' do
    # ...
  end
end
```